The test clang/test/Parser/cuda-check-input-kind-assoc.cuh is never run

In a local build, if I run:

`<builddir>/bin/llvm-lit <srcdir>/clang/test/Parser/cuda-check-input-kind-assoc.cuh`

I get:

```console
llvm-lit: llvm/utils/lit/lit/llvm/config.py:506: note: using clang: build/bin/clang
llvm-lit: llvm/utils/lit/lit/discovery.py:276: warning: input 'src/clang/test/Parser/cuda-check-input-kind-assoc.cuh' contained no tests
error: did not discover any tests for provided path(s)
```

I think this suggests this test case is never actually run.

--- a/clang/test/lit.cfg.py
+++ b/clang/test/lit.cfg.py
@@ -33,6 +33,7 @@ config.suffixes = [
     ".m",
     ".mm",
     ".cu",
+    ".cuh",
     ".hip",
     ".hlsl",
     ".ll",

